openshift/templates/images/postgres/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM postgres:15.4-bullseye
+# OpenShift uses version from buildconfig - Don't update
```

Comment:
what does this mean?
what buildconfig is that?
how does OpenShift use that version if it's shown in this file?

Reply:
this is an education piece, on how build configs, to one of your previous comments, why is this postgres15.5-bullseye here.... this replaces the from image... so technically in this dockerfile you could say something like `FROM scratch`